packages/oidc-dpop-client-browser/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "@inrupt/oidc-dpop-client-browser",
+  "version": "0.0.1",
+  "description": "A module extending oidc-client-js with new features, such as dynamic client registration and DPoP support.",
+  "homepage": "https://github.com/inrupt/solid-client-authn-js/packages/oidc-dpop-client-browser/",
```

I think this is what you meant:

```suggestion
  "homepage": "https://github.com/inrupt/solid-client-authn-js/tree/master/packages/oidc-dpop-client-browser/",
```

That said, maybe it's better to just not include the `homepage` property, since there's not much more there to find.

---

Yeah I wasn't sure if it should be included or not. I ended up including it and having the target file link back to the root of the repository, so that one could get context, but since `oidc-dpop-client-browser` isn't really meant for wide public adoption anyway maybe I'll just remove it.